apps/test/unit/templates/sectionProgress/sectionStandardsProgressReduxTest.jsx

```diff
@@ -0,0 +1,134 @@
+import {expect} from 'chai';
```

Comment:
I also created this file in https://github.com/code-dot-org/code-dot-org/pull/33012/files (with .js instead of .jsx extension). So we should decide which extension we want to use and just a heads up on merge conflicts.

Reply:
Thanks for the heads up, Dani! I consolidated into you js file.